CHANGELOG.md

```diff
@@ -12,8 +12,10 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
 
 ##### Bug Fixes
 
-* None.  
-
+* Headers from vendored frameworks no longer end up in the HEADER_SEARCH_PATH when 
+  using frameworks. They are assumed to be already present as modular headers in the
+  framework itself.  
+  [Mark Spanbroek](https://github.com/markspanbroek)
```

Comment:
Please add a reference to either #4805 or this PR, which ever of both you think describe better what was the issue and what had to be done. While we tend to prefer adding the issue, if it can be considered "canoncial" / very descriptive for the problem and there are not a bunch of duplicates around it, which don't really describe the root of the problem.